dashboard/app/main.go

```diff
@@ -840,25 +848,25 @@ func handleBug(c context.Context, w http.ResponseWriter, r *http.Request) error
 	// - no fix bisections have been performed on the bug
 	// - fix bisection was performed but resulted in a crash on HEAD
 	// - there have been infrastructure problems during the job execution
-	if bug.BisectFix == BisectNot {
-		fixBisections, err := loadBisectionsForBug(c, bug, JobBisectFix)
+	if len(fixBisections.all()) > 1 || len(fixBisections.all()) > 0 && bisectFix == nil {
+		uiList, err := fixBisections.uiAll(c)
 		if err != nil {
 			return err
 		}
-		if len(fixBisections) != 0 {
+		if len(uiList) != 0 {
 			data.Sections = append(data.Sections, makeCollapsibleBugJobs(
-				"Fix bisection attempts", fixBisections))
+				"Fix bisection attempts", uiList))
 		}
 	}
 	// Similarly, a cause bisection can be repeated if there were infrastructure problems.
-	if bug.BisectCause == BisectNot {
-		causeBisections, err := loadBisectionsForBug(c, bug, JobBisectCause)
+	if len(causeBisections.all()) > 1 || len(causeBisections.all()) > 0 && bisectCause == nil {
```

Comment:
len(causeBisections.all()) > 0 check in this case checks for len(causeBisections.all()) == 1

Reply:
The objectives were as follows:
a) If there are more than 1 bisection, always show the list.
b) Otherwise only show it if we didn't pick up one bisection to separately show it on the page (`bisectCause == nil`).